chia/server/start_wallet.py

```diff
@@ -55,7 +58,14 @@ def service_kwargs_for_wallet(
         connect_peers = []
         node.full_node_peer = None
     network_id = service_config["selected_network"]
-    kwargs = dict(
+    rpc_port = service_config.get("rpc_port")
+    rpc_info: RpcInfo = None
+    if rpc_port is not None:
+        rpc_info = (WalletRpcApi, service_config["rpc_port"])
+
+    return Service(
+        advertised_port=service_config["port"],
```

Comment:
```suggestion
        advertised_port=service_config["port"],
```

nit: You added it at the end in an other place.

Reply:
nit: Who cares since I pass by name instead of position.  `:]`  But seriously, yep, will change.  de7535c9e026526cb13e856c0102ed13c5778186